src/v3/src/transformer/phone/phoneChallengeTransformer.ts

```diff
@@ -50,7 +50,7 @@ export const transformPhoneChallenge: IdxStepTransformer = ({ transaction, formB
   const sendInfoText = smsMethodType
     ? loc('oie.phone.verify.sms.codeSentText', 'login')
     : loc('mfa.calling', 'login');
-  const phoneNumberSpan = phoneNumber ? `<span class="strong no-translate">${phoneNumber}.</span>` : null;
+  const phoneNumberSpan = phoneNumber ? `<span class="strong no-translate">&lrm;${phoneNumber}.</span>` : null;
```

Comment:
Is there any difference between this and using `dir` on the `span` tag? If the only thing is the way it handles the period, that could be placed outside of the span tag that has `dir` set.

Reply:
yeah it looks like the only difference is the way it handles the period.  Ill add the `dir` attr and place the period outside

Reply:
Can you to add a comment above the places where you added the unicode character with a short note about why it's there and a link to the w3c docs?